lib/brakeman/util.rb

```diff
@@ -124,6 +124,18 @@ def hash_access hash, key
     nil
   end
 
+  def call_has_param call, key
```

Comment:
This is not related to the return value of any method. It's a syntax error because `return` has different precedence rules. Just remove `return`.